.github/workflows/licensed.yml

```
@@ -16,9 +16,12 @@ jobs:
       - uses: actions/checkout@v3
         with:
           fetch-depth: 0 # prefer to use a full fetch for licensed workflows
-      # https://github.com/jonabc/setup-licensed/releases/tag/v1.1.1
-      - uses: jonabc/setup-licensed@82c5f4d19e8968efa74a25b132922382c2671fe2
+      - uses: ruby/setup-ruby@v1
```

Can you update this to pin to a commit SHA?

https://docs.github.com/en/actions/security-guides/security-hardening-for-github-actions#using-third-party-actions

---

Sure, I can take a look. I'm assuming it's not required for the `github/setup-licensed` one though? I'm guessing that's similar to the `actions/*` ones that aren't pinned

---

Not the latest, I just grabbed it from code-searching this org 😉 
```suggestion
      - uses: ruby/setup-ruby@8575951200e472d5f2d95c625da0c7bec8217c42 # v1.161.0
```